### DISCUSSION TITLE
GitHub actions is not able to fetch GitHub package

### DISCUSSION THREAD

**Discussion by dharmesh-hemaram** [community/community#39935] [Actions] [answered]
Hi Team,

I have two repo core-common and core-extension under Dhruv-techappa organization.

Using GitHub action I published core-common package to GitHub packages.

I'm able to use same in my local by creating PAT with read:packages permission.

When I'm running GitHub actions in core-extension and added core-common as dependency.

It throws unauthorised error.

GitHub action in core extension 
----------
name: ESLint

on:
  push:
    branches: [ "master", "develop" ]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: [ "master", "develop"]

jobs:
  eslint:
    name: Run eslint scanning
    runs-on: ubuntu-latest
    permissions:
      contents: read
      security-events: write
      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
    steps:
      - uses: actions/checkout@v3
      - name: Use Node.js ${{ matrix.node-version }}
        uses: actions/setup-node@v3
        with:
          always-auth: true
          node-version: 16
          registry-url: https://npm.pkg.github.com
          scope: '@dhruv-techapps'
          cache: 'npm'      
        
        
      - name: Install Dependencies
        run: |
          npm install eslint@8.10.0
          npm install @microsoft/eslint-formatter-sarif@2.1.7
          env:
            NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
-----------

I have added core-extension as repo under core-common package

[core-extension](https://github.com/Dhruv-Techapps/core-extension)

[core-common](https://github.com/Dhruv-Techapps/core-common)

**Comment by dharmesh-hemaram** [comment]
Ref to GitHub action running status [status](https://github.com/Dhruv-Techapps/core-extension/actions/runs/3539135746/jobs/5940701798)

**Comment by ossanna16** [comment, marked as answer]
Hi there @dharmesh-hemaram and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
